packages/documentation/src/content/docs/integration/deployment/docker-compose.mdx

```diff
@@ -96,14 +96,14 @@ The example domain and subdomain values are for demonstration purposes only. You
 Update the variables in the following compose file with values relevant to your environment and system. Refer to the [environment variables page](/resources/environment-variables/) for details.
 
 :::note[Update compose file]
-You must change the values enclosed within curly brackets {} in the following compose file.
+You must change the values enclosed within curly brackets {} in the following compose file. Substitute `newest-version` with the latest Rafiki <LinkOut href='https://github.com/interledger/rafiki/releases'>version</LinkOut>.
```

Comment:
`newest-version` isn't in curly brackets, it's in `< >`. Do you think that could confuse anyone?

Reply:
Good point. I removed the curly reference.